0.1.0/lib.typ

```diff
@@ -0,0 +1,159 @@
+#import "@preview/acrostiche:0.3.1": init-acronyms, print-index
+#import "lib/research-questions.typ": init-rqs
+
+// workaround for lack of std scope
+#let std-biblio = bibliography
+
+// store in states such that these need to be passed
+// only once, in the `thesis` function
+#let _authors = state("authors", ())
+#let _title = state("title", [])
+#let _supervisors = state("supervisors", ())
+#let _year = state("year", [2023-2024])
+
+#let thesis(
```

Comment:
Don't know if it's useful to have default values for the parameters here. These should all be overridden anyway (except for year, look at previous comment).

Reply:
You're right, but to keep these arguments named instead of positional, we should maybe have `none` as default value.
Using this function without named arguments would become messy, I think.